### ISSUE TITLE
Issue with inline image attachement

### ISSUE THREAD

**Issue by irmantus** [roundcube/roundcubemail#9476]
### Prerequisites

- [X] I have [searched](https://github.com/roundcube/roundcubemail/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I can recreate the issue with all plugins disabled

### Describe the issue

When I send email with inline image to Gmail it gets delivered as plain html/base64 tag:
![image](https://github.com/roundcube/roundcubemail/assets/37515045/fd80c1bf-a9e9-4d6f-9aef-cb701dca6d09)

### What browser(s) are you seeing the problem on?

Other

### What version of PHP are you using?

8.1

### What version of Roundcube are you using?

1.6.7

**Comment by pabzm** [comment]
Thank you for the report!

Could you please describe what actually is the problem is? Sending images inline as data-URIs is not a bug but a feature, as far as I know.

**Comment by alecpl** [comment]
No, the images are supposed to be extracted to a separate inline attachment part.

**Comment by alecpl** [comment]
It does not seem like we have enough info here. I'm unable to reproduce.
